clap_markdown/src/lib.rs

```diff
@@ -99,10 +99,11 @@ impl Markdown {
         if subcommands.peek().is_some() {
             let _ = writeln!(markdown, "\n### Commands\n");
         }
+        let mut subcommands: Vec<_> = subcommands.collect();
+        subcommands.sort_by_key(|cmd| cmd.get_name());
         for cmd in subcommands {
-            let name = cmd.get_name();
-            let human_path = format!("{} {name}", parents.join(" "));
-            let link_path = format!("{}_{name}", parents.join("_"));
+            let human_path = format!("{} {cmd}", parents.join(" "));
+            let link_path = format!("{}_{cmd}", parents.join("_"));
```

Comment:
hmmm...I'm not sure either. I thought I did a screen share session with you to figure out this bit of code...but that was long enough ago that I don't really remember. It will be changed.